sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -179,9 +178,9 @@ def s3_event_handler(event):
     ingestion_date_object = get_file_creation_date(s3_filepath)
 
     file_params["ingestion_date"] = ingestion_date_object
-    with db.Session() as session:
-        update_file_catalog_table(session, file_params)
-    logger.info("Wrote data to file catalog table")
+    with db.Session() as session, session.begin():
+        session.add(models.ScienceFiles(**file_params))
```

Comment:
that's better!

Reply:
Can you add TODO in here that we will need to call different table based on which files we ingest?

Reply:
I made an issue which I prefer to adding TODOs in the code: https://github.com/IMAP-Science-Operations-Center/sds-data-manager/issues/350